File: tests/corpus/positive-fixtures/QA-SE-002/InventoryTest.cs
using System.Threading;
using OpenQA.Selenium;
using OpenQA.Selenium.Chrome;

class InventoryTest
{
    public void LoadInventory(IWebDriver driver)
    {
        driver.Navigate().GoToUrl("https://app.example.com/inventory");
        Thread.Sleep(3000);
        driver.FindElement(By.Id("inventory-table"));
    }
}

File: src/tests/corpus/positive-fixtures/QA-SE-001/InventoryTest.java
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.chrome.ChromeDriver;

class InventoryTest {
    void loadInventory(WebDriver driver) {
        driver.get("https://app.example.com/inventory");
        try {
            Thread.sleep(3000);
        } catch (InterruptedException e) {
            throw new RuntimeException(e);
        }
        driver.findElement(By.id("inventory-table"));
    }
}
